lib/configGenerator/generateAnsibleConfig.js

```diff
@@ -114,6 +115,11 @@ async function generateAnsibleConfig(
   // Governance proposals
   config.governance_proposal_count = 2;
 
+  // Mixer variables
+  config.mix_amount = 100;
+  config.remix_amount = 30;
+  config.coinjoin_wallet_name = "{{ coinjoin_wallet_name }}";
```

Comment:
This results in the config YAML file containing a Jinja2 template like `coinjoin_wallet_name: '{{ coinjoin_wallet_name }}'`, but this file is not templated. How is this supposed to work? Why do we need to be able to configure the wallet name?

Reply:
Woah okay that answers my previous question... that should be an actual value... :)

I think a search and replace probably went wrong, fixed